tests/e2e/config/bootstrap.js

```diff
@@ -167,11 +167,31 @@ function observeConsoleLogging() {
 			text.startsWith( 'Failed to decode downloaded font:' ) ||
 			text.startsWith( 'OTS parsing error:' ) ||
 			text.includes( 'Download the React DevTools for a better development experience' ) ||
-			text.includes( 'https://fb.me/react-unsafe-component-lifecycles' )
+			text.includes( 'Can\'t perform a React state update on an unmounted component' ) ||
+			text.includes( 'https://fb.me/react-unsafe-component-lifecycles' ) ||
+			text.includes( 'https://fb.me/react-strict-mode-' )
```

Comment:
Some of these are triggered by Gutenberg itself and are good to know about but not something we should be expecting in our tests and by extension require our tests to fail if they weren't called.